packages/@uppy/dashboard/types/index.d.ts

```diff
@@ -65,7 +65,7 @@ export interface DashboardOptions extends Options {
   theme?: 'auto' | 'dark' | 'light'
   trigger?: string
   width?: string | number
-  autoOpenFileEditor?: boolean
```

Comment:
The other type still needs to exist but with `/** @deprecated use autoOpen instead */`